kong/db/schema/entities/routes_subschemas.lua

```diff
@@ -67,17 +67,39 @@ local grpc_subschema = {
 
 
 if kong and kong.configuration and  kong.configuration.router_flavor == "expressions" then
-  return {}
-
-else
-  return {
-    http = http_subschema,  -- protocols is the subschema key, and the first
-    https = http_subschema, -- matching protocol name is selected as subschema name
-    tcp = stream_subschema,
-    tls = stream_subschema,
-    udp = stream_subschema,
-    tls_passthrough = stream_subschema,
-    grpc = grpc_subschema,
-    grpcs = grpc_subschema,
-  }
+
+  -- now http route in flavor 'expressions' accepts 'sources' 'and destinations'
+
+  http_subschema.fields[1] = nil  -- sources
+  http_subschema.fields[2] = nil  -- destinations
```

Comment:
We should add assertions here to ensure we are deleting expected fields.  We might insert some new fields in the future and forget to correct the index.

Reply:
Yes, we should do it. But `http_subschema.fields` is an array, not a hash table, do we have some easy way to check it?

Reply:
Maybe something like this:
``` lua
assert(type(http_subschema.fields[1].sources) == "table", "<message>")
```

Reply:
done.